### ISSUE TITLE
Cache Spec being ignored

### ISSUE THREAD

**Issue by vaamyob** [bazaarvoice/dropwizard-configurable-assets-bundle#33]
Latest release dropwizard-configurable-assets-bundle, and 0.8.0 RC

yaml:

    assets:
        cacheSpec: maximumSize=0
        overrides:
            /lb: /opt/lb/

Service:

    bootstrap.addBundle(new ConfiguredAssetsBundle("/assets/app/",
                               CacheBuilderSpec.parse("maximumSize=0"), "/", "index.html", "assets"));

I just want to disable the cache, for testing purposes, but neither change has any effect.
If I delete the `index.html` file from the `/opt/lb` directory, `curl http://localhost:8080/lb/index.html` still loads the cached version.

**Comment by vaamyob** [comment]
So this was closed because a workaround was found?

**Comment by nbauernfeind** [comment]
This issue is not closed. The other issue is closed because he had misconfigured his assets bundle.

I haven't had a chance to look into upgrading. I've been secretly hoping that dropwizard will do a full release so I only have to go messing about once. Also, I left Bazaarvoice a while ago and no longer have the ability to merge / release thing anymore.

I've toyed around with tossing the dropwizard bundles into a separate organization that I can work from; but BV employees asked for an opportunity to manage these themselves before I do that.

**Comment by nbauernfeind** [comment]
Actually, I'll toy around this weekend and get you a released version of this. It'll have a different maven group id though.

**Comment by vaamyob** [comment]
I understand.  Whatever light you can shed on the situation would be appreciated.

**Comment by nbauernfeind** [comment]
Sorry for the super delayed response to this. @amarjeetsingh12 was absolutely correct about the behavior being incorrect. This has been fixed here: https://github.com/dropwizard-bundles/dropwizard-configurable-assets-bundle/commit/e676c80a186325b7374cc16e7de8048fab42ac9e

I've released a version for dropwizard 0.8.0 and it should be making its way to maven central. Please note that the group id has changed (and that I can no longer close this issue).

```xml
<dependency>
  <groupId>com.nefariouszhen.dropwizard</groupId>
  <artifactId>dropwizard-configurable-assets-bundle</artifactId>
  <version>0.8.0</version>
</dependency>
```

You can see the changes and the repository here:
https://github.com/dropwizard-bundles/dropwizard-configurable-assets-bundle

**Comment by vaamyob** [comment]
Excellent!  I'll check it out.  Thanks!
